### ISSUE TITLE
False-positive for directive-selector-name

### ISSUE THREAD

**Issue by CSchulz** [palantir/tslint#1682]
### Bug Report

- __TSLint version__: `3.15.1`
- __TypeScript version__: `2.0.2`
- __Running TSLint via__: CLI / IDEA

#### TypeScript code being linted

```ts
@Directive({
    selector: '[past][formControlName],[past][formControl],[past][ngModel]',
    providers: [{
        provide: NG_VALIDATORS,
        useExisting: forwardRef(() => DatePastValidator),
        multi: true,
    }],
    host: {'[attr.date]': 'date? "" : null'},
})
export class DatePastValidator implements Validator {
```

with `tslint.json` configuration:

```json
{
  "rulesDirectory": [
    "node_modules/vrsource-tslint-rules/rules",
    "node_modules/codelyzer"
  ],
  "extends": "tslint:latest",
  "rules": {
    "directive-selector-name": [
      true,
      "camelCase"
    ]
  }
}
```

#### Actual behavior

```
> tslint ./src/**/*.ts ./test/**/*.ts -s node_modules/tslint-stylish -t stylish

src/app/form/fields/date-past.validator.ts
10:15  directive-selector-name  The selector of the directive "DatePastValidator" should be named camelCase (https://goo.gl/QS8kEs)
```

#### Expected behavior

The selector name should pass the linting.

**Comment by adidahiya** [comment]
Please file this issue in the custom rules project, in this case I think that's codelyzer. That rule doesn't exist in this repository.

**Comment by CSchulz** [comment]
Upps, my fault!

see https://github.com/mgechev/codelyzer/issues/15
